app/helpers/companies_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module CompaniesHelper
```

Comment:
delete, we are not using helper

are you using scaffolding? if yes, please, stop doing this

Reply:
deleted

Reply:
I used `rails g controller`